src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -160,6 +161,17 @@ export const roomCreateSchema = Yup.object().shape<RoomSchemaShape>({
       .required("Url is required!")
       .min(3, ({ min }) => `Url must be at least ${min} characters`),
   }),
+  image_url: Yup.string().required("Room image is required"),
+});
+
+export const roomEditSchema = Yup.object().shape<RoomSchemaShape>({
+  title: Yup.string()
+    .required("Room name is required")
+    .min(3, ({ min }) => `Name must be at least ${min} characters`),
+  url: Yup.string()
+    .required("Url is required!")
+    .min(3, ({ min }) => `Url must be at least ${min} characters`),
+  image_url: Yup.string().required("Room image is required"),
 });
```

Comment:
Is there a reason why we can't reuse the same schema stuff that is used on creation? What differs between the two? And why does it differ? Feels like they should both validate with the same code?

Reply:
When creating a room there is an option to select either new venue and room or simple room with url. That validation fails in  the edit because that field doesn't exist anywhere in the firebase.

Reply:
Fair enough. I'm not deeply familiar with yup, but is there a way we can share the same partial validation rules between the 2? It looks like it's just an object, so perhaps we can move the object outside of the `.shape()` call, and then use the common bits from that to build the 2 separate validations.

Reply:
I created some constants that are being shared between them.